modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-respect-rbac-using-gitops_{context}"]
+= Configuring respectRBAC using {gitops-title}
+
+The `respectRBAC` configuration feature in Argo CD controls how Argo CD watches resources on a cluster. By default, Argo CD attempts to watch all Kubernetes resources (CRDs) on a cluster at the cluster scope. You can restrict the ArgoCD controller from discovering or syncing specific resources using only controller RBAC, without manually configuring resource exclusions.
+
+To enable this feature, set the `resource.respectRBAC` key in the `argocd-cm` config map. After you set this key, the controller automatically stops watching resources it cannot list or access. For example, if the Argo CD cluster role prevents Argo CD from watching OpenShift Routes, it returns an error during synchronization, stating it cannot watch the resource.
+
+[NOTE]
+====
+You can only enable `respectRBAC` by modifying the `argocd-cm` config map. If a user does not have permission to edit the config map, they cannot enable this feature. The `respectRBAC` feature of the {gitops-shortname} Operator allows users to update `respectRBAC` configurations in the config map through the Argo CD resource.
+====
+
+You can enable the `respectRBAC` feature by creating an Argo CD instance in the CLI or the Web UI. 
+
+.Pre-requisites
+
+Ensure that you created a namespace and updated it in the *Subscription* resource, so it can host a cluster-scoped Argo CD instance. For more information, see Using an Argo CD instance to manage cluster-scoped resources.
+
+[id="configuring-respectRBAC-using-the-cli_{context}"]
+== Configuring respectRBAC using the CLI
+
+You can configure `respectRBAC` in the CLI.
+
+.Procedure
+
+. Create a YAML object file, for example, `openshift-gitops-sub.yaml`, to configure the `respectRBAC` feature using the CLI. 
```

Comment:
Don't use a YAML file named `*-sub.yaml` if it doesn't contain a subscription.

Reply:
Discussed further with @jgwest and he has suggested us to use a file name like this **argo-cd-resource.yaml**.

Reply:
Fixed.